cloudinit/net/sysconfig.py

```diff
@@ -267,6 +273,11 @@ def copy(self, copy_children=False, copy_routes=False):
             c.routes = self.routes.copy()
         return c
 
+    def str_skipped(self, key, val):
+        if key == 'TYPE' and val == 'Vlan':
+            return True
+        return False
```

Comment:
We can be a little more terse here if we want:

```suggestion
        return key == 'TYPE' and val == 'Vlan'
```

or even

```suggestion
        return (key, value) == ('TYPE', 'Vlan')
```

This is a nit, I'll leave it up to you if you want to apply either of these.

Reply:
do not underestimate the power of a clearly written out `if` on the readability

Reply:
I'm gonna go with @igalic here and opt for the bigger and clearer code. :+1: